3p/iframe-messaging-client.js

```diff
@@ -25,6 +25,8 @@ export class IframeMessagingClient {
   constructor(win) {
     /** @private {!Window} */
     this.win_ = win;
+    /** @private {?string} */
+    this.rtvVersion_ = (win.AMP_CONFIG && win.AMP_CONFIG.v) || null;
```

Comment:
And of course you'll need to ensure the message after amp-<version> does not start with a number (which appears to be the case give you're calling JSON.stringify on the message